### ISSUE TITLE
Check first argument of NamedTuple/namedtuple

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#4589]
Mypy doesn't complain about this code, even though the name arguments to `NamedTuple` and `namedtuple` are inconsistent with the assignment targets:

```py
from typing import NamedTuple
from collections import namedtuple

A = NamedTuple('X', [('a', int)])  # First argument should be 'A'?
B = namedtuple('X', ['a'])  # First argument should be 'B'?
```

I'd expect mypy to complain about the `'X'` arguments in both cases.

**Comment by ilevkivskyi** [comment]
By the way I have noticed this kind of mistake several times during last week, so it may be important to fix.
